method_workflows/PR_review_guide.md

```diff
@@ -23,8 +23,8 @@ Refer to the [specifications][out-specs].
 
 ## Check that steps to run methods/pre-processing are wrapped in Dockerfiles
 
-- Should be uploaded to apaeval team Dockerhub, not on personal accounts 
-(see [execution workflow README on Containers][ewf-readme-container])
+- Should be uploaded to [biocontainers][biocontainers], not on personal accounts 
+(see [method workflow README on Containers][mwf-readme-container])
```

Comment:
Link was broken for me

Reply:
hmmm, can't reproduce that. It's working for me

Reply:
Links are working for me too.

Reply:
Hmm, the biocontainers link works fine but the 'method workflow README on Containers' link still gives me this error. Anyway if it's just me not a big deal.


<img width="782" alt="image" src="https://github.com/iRNA-COSI/APAeval/assets/49978382/fcc1cb96-fe29-4eef-b00e-c012a4aa2f6c">